webservices/legal_docs/load_legal_docs.py

```diff
@@ -114,6 +114,12 @@ def index_advisory_opinions():
         es = utils.get_elasticsearch_connection()
 
         logger.info("getting citations...")
+        ao_names_results = db.engine.execute("""SELECT ao_no, name FROM aouser.document
```

Comment:
IMO, the enclosing function does a lot and should be refactored into several functions. I realize that refactoring would be greater than the scope of this pull request, and won't hold up the merge for this. Will you consider it?